wisdom-war/src/components/sections/Cards.tsx

```diff
@@ -1,8 +1,16 @@
 import React from "react";
+import Button from "../buttons/Button";
+import { useNavigate } from "react-router-dom";
 
 type CardProps = { quizName: string; description: string; level: string };
 
 function Card(props: CardProps) {
+  const navigate = useNavigate();
```

Comment:
Now we have to delete the navigation, and use Khalo's path.
Fix the buttons to make them work between the pages.